lib/logger/utils.ts

```diff
@@ -49,6 +51,47 @@ const contentFields = [
   'yarnLockParsed',
 ];
 
+export default function prepareError(err: Error): Record<string, unknown> {
+  const response: Record<string, unknown> = {
+    ...err,
+  };
+
+  // Can maybe removed?
+  if (!response.message && err.message) {
+    response.message = err.message;
+  }
+
+  // Can maybe removed?
+  if (!response.stack && err.stack) {
+    response.stack = err.stack;
+  }
+
+  // handle got error
+  if (err instanceof RequestError) {
+    const options: Record<string, unknown> = {
+      headers: clone(err.options.headers),
+      url: err.options.url?.toString(),
+    };
+    response.options = options;
+
+    for (const k of ['username', 'password', 'method', 'http2']) {
+      options[k] = err.options[k];
```

Comment:
Will they still be sanitized correctly?

Reply:
sure, the error simply got patched before sanitizing, maybe we can remove the errSerializer entirely later.